test/functional/p2p_segwit.py

```diff
@@ -1364,7 +1365,11 @@ def test_segwit_versions(self):
         assert_equal(len(self.nodes[1].getrawmempool()), 0)
         for version in list(range(OP_1, OP_16 + 1)) + [OP_0]:
             # First try to spend to a future version segwit script_pubkey.
-            script_pubkey = CScript([CScriptOp(version), witness_hash])
+            if version == OP_1:
+                # Don't use 32-byte v1 witness (used by taproot)
```

Comment:
Unify Taproot capitalization in comments.
```suggestion
                # Don't use 32-byte v1 witness (used by Taproot)
```

Reply:
Updated.